rotkehlchen/assets/exchanges_mappings/poloniex.py

```diff
@@ -140,8 +140,15 @@
     evm_address_to_identifier('0xb3999F658C0391d94A37f7FF328F3feC942BcADC', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'HFT',  # noqa: E501
     evm_address_to_identifier('0xbB95cc1c662D89822bda29D2e147B124406e6e42', ChainID.BINANCE, EvmTokenKind.ERC20): 'TRR',  # noqa: E501
     evm_address_to_identifier('0xaaAEBE6Fe48E54f431b0C390CfaF0b017d09D42d', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'CEL',  # noqa: E501
-    evm_address_to_identifier('0x8dAEBADE922dF735c38C80C7eBD708Af50815fAa', ChainID.BINANCE, EvmTokenKind.ERC20): 'ALPACA',  # noqa: E501
+    evm_address_to_identifier('0x8dAEBADE922dF735c38C80C7eBD708Af50815fAa', ChainID.BINANCE, EvmTokenKind.ERC20): 'TBTC',  # noqa: E501
```

Comment:
Why did this change to TBTC? This seems wrong. That's not a contract in BSC

https://bscscan.com/address/0x8dAEBADE922dF735c38C80C7eBD708Af50815fAa

Reply:
Saw that TBTC was missing and when adding it the address was already there but with the wronmg symbol. ALpaca is a few lines later

Reply:
Please check the link of the contract. That's not contract. Address is wrong.